src/main/resources/javascript/locales/de.json

```diff
@@ -260,6 +260,8 @@
         "publishMediumPublished": "Veröffentlicht - {{language}}",
         "publishShort": "Veröffentlichen",
         "publishShortPublished": "Veröffentlicht",
+        "publishPublishedMedium": "Veröffentlicht - {{language}}",
+        "publishPublishedShort": "Veröffentlicht",
```

Comment:
Are these needed? Looks to be duplicates of `publishMediumPublished` and `publishShortPublished` which should be what were using atm. (These were switched from this PR)

If it's breaking, I think we need to open a ticket to switch to the latter.

If anything, these keys also do not exist in en.json but it does in fr.json, so I think we should remove the fr ones.

Reply:
I removed them from fr and de